src/schema/event/utils.ts

```diff
@@ -0,0 +1,31 @@
+import queryBuilder from '../../utils/queryBuilder';
+import { EventParams } from './types';
+
+export const buildEventQuery = (params: EventParams) => {
```

Comment:
I first thought that this is query builder for single event. Maybe the name could be something like `buildEventListQuery`